Assets/MixedRealityToolkit.Examples/AssemblyInfo.cs

```diff
@@ -3,8 +3,8 @@
 
 using System.Reflection;
 
-[assembly: AssemblyVersion("2.3.0.0")]
-[assembly: AssemblyFileVersion("2.3.0.0")]
+[assembly: AssemblyVersion("2.4.0.0")]
```

Comment:
Was this branch built on mrtk_dev but merged into stabilization? Looks like there are some dev branch only changes that were included.

Reply:
Yes this branch was based on mrtk_dev, the version number was reverted to 2.3 from 2.4 in #7049